src/Core/Router.php

```diff
@@ -39,13 +43,13 @@ private function direct(string $uri, string $methodType)
     private function callAction(string $controller, string $action)
     {
         $controller = "TrendingRepos\\Controller\\{$controller}";
```

Comment:
By the way, `$contoller` variable here means something else than below, right? Therefore, it deserves different name, no reason to override it on the next line.